community/kernel/src/main/java/org/neo4j/kernel/impl/api/store/StorePropertyCursor.java

```diff
@@ -302,77 +279,64 @@ private boolean nextBlock()
         // Read remaining data from previous property (if it was not read)
         while ( remainingBlocksToRead-- > 0 )
         {
+            // don't really care to retry since we discard the data...
```

Comment:
why not to do: `cursor.setOffset(offset + remainingBlocksToRead * Long.BYTES)` ?